registry/remote/example_test.go

```diff
@@ -451,6 +479,51 @@ func ExampleRepository_Fetch_artifactReferenceManifest() {
 	// {"mediaType":"application/vnd.cncf.oras.artifact.manifest.v1+json","artifactType":"example/signature","blobs":null,"subject":{"mediaType":"application/vnd.oci.image.manifest.v1+json","digest":"sha256:00e5ffa7d914b4e6aa3f1a324f37df0625ccc400be333deea5ecaa199f9eff5b","size":24}}
 }
 
+// ExampleRepository_Fetch_blobsOfArtifactManifest gives an example of pulling the blobs
+// of an artifact manifest.
+func ExampleRepository_Fetch_blobsOfArtifactManifest() {
+	repo, err := remote.NewRepository(fmt.Sprintf("%s/%s", host, exampleRepositoryName))
+	if err != nil {
+		panic(err)
+	}
+	ctx := context.Background()
+
+	// Fetch the artifact manifest by digest. The digest can be obtained from Referrers API.
+	exampleDigest := "sha256:73bccdadf23b5df306bf77b23e1b00944c7bbce44cf63439afe15507a73413b5"
+	_, rc, err := repo.FetchReference(ctx, exampleDigest)
+	if err != nil {
+		panic(err)
+	}
+	defer rc.Close()
+
+	// Parse the pulled manifest and fetch its blobs.
+	pulledContent, err := io.ReadAll(rc)
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println(string(pulledContent))
```

Comment:
We need to verify the size and digest of the pulled content before this line. See https://github.com/oras-project/oras-go/blob/main/registry/remote/example_test.go#L472-L475

Reply:
Added the verification.